src/components/ListChildrenCard/ListChildrenCard.js

```diff
@@ -7,7 +8,10 @@ export default function ListChildrenCard(props) {
         <>
             {childrenCards.map(card => {
                 return(
-                    <div>{card.title}</div>
+                    <Tooltip title={card.status} placement="right">
+                        <span>{card.title}</span>
+                        <br />
+                    </Tooltip>
```

Comment:
@DmitryApan https://prnt.sc/ujf07q please configure every list to have unique color and show square or circle with appropriate background color and tooltip near children task